package.json

```diff
@@ -17,7 +17,7 @@
   "devDependencies": {
     "@babel/plugin-proposal-class-properties": "^7.13.0",
     "@mate-academy/eslint-config": "*",
-    "@mate-academy/scripts": "^0.9.14",
+    "@mate-academy/scripts": "^1.2.8",
```

Comment:
In the future - Do not add extra files to PR
To add the file to a commit use `git add src/.` & `git commit -m 'your message'`